tools/interactive/interactivetool_ml_jupyter_notebook.xml

```diff
@@ -135,12 +136,12 @@ except FileNotFoundError:
         <conditional name="mode">
             <param name="mode_select" type="select" label="Do you already have a notebook?" help="Select a set of default notebooks or load an existing one.">
                 <option value="scratch">Start with default notebooks</option>
-                <option value="github">Start with a GitHub repository</option>
+                <option value="pull_repo">Start with a code repository</option>
                 <option value="previous">Load an existing notebook</option>
             </param>
             <when value="scratch"/>
-            <when value="github">
-                <param name="github_url" type="text" value="" label="GitHub URL" optional="False"/>
+            <when value="pull_repo">
+                <param name="repo_url" type="text" value="" label="Online code repository (Git-based) URL" optional="False" help="Provide URL of Git-based code repository (for example, GitHub repository URL)" />
```

Comment:
maybe disallow whitespaces?

Reply:
> Do you have a nice example github repo that you can mention here?

Add this one: https://github.com/anuprulez/gpu_jupyterlab_ct_image_segmentation

Reply:
> maybe disallow whitespaces? 

Here I fixed: `GitHub repository URL)" />` to `GitHub repository URL)"/>`